packages/react-core/src/components/Chip/Chip.tsx

```diff
@@ -139,12 +139,13 @@ class Chip extends React.Component<ChipProps, ChipState> {
       tooltipPosition,
       component,
       ouiaId,
+      textMaxWidth,
       ...props
     } = this.props;
     const Component = component as any;
     return (
       <Component
-        {...(this.props.textMaxWidth && {
```

Comment:
This was causing a `textmaxwidth` attribute to be rendered due to the spread `...props` on line 154